ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2129,3 +2129,46 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-merge-robot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
+    gcs_configuration:
+      bucket: origin-ci-private
+      job_url_prefix: https://deck-internal-ci.apps.ci.l2s4.p1.openshiftapps.com/view/
+      mediaTypes:
+        log: text/plain
+    gcs_credentials_secret: gce-sa-credentials-gcs-private
+  hidden: true
+  interval: 10m
```

Comment:
Looks a bit excessive... I think a 12h period would be fine? Is it a problem if this has a day of latency?

Reply:
I agree with Petr. How often the LDAP relations will change. I would be fine to run the job even once per day.

Reply:
Changed to 24h.